### DISCUSSION TITLE
array多层级嵌套时，子集怎么依赖父集

### DISCUSSION THREAD

**Discussion by 18319700115** [alibaba/formily#3625] [Q&A 2.x] [answered]
代码片段：https://codesandbox.io/s/hardcore-https-iyynhf?file=/App.tsx:583-588
需求：price需要依赖total的变化而变化
描述：单个array的时候，子集price的依赖路径path可以是...total，但是嵌套两个array的时候就不生效了，求教一下这个路径要怎么写

**Comment by 18319700115** [comment]
除了https://github.com/alibaba/formily/discussions/1922和完全匹配路径写法，还有别的写法吗

**Comment by janryWang** [comment, marked as answer]
有parent属性

**Reply by 18319700115** [reply]
具体怎么使用呢
